### PR TITLE
dbsp: Add circuit diagram to trace operator.

### PR DESCRIPTION
**Pull request by blp**
Possibly obvious to others but not to me.

Is this a user-visible change (yes/no): no

### CODE REVIEW

File: crates/dbsp/src/operator/trace.rs [hunk 1]
```diff
@@ -235,6 +235,16 @@ where
         B: BatchReader<Time = ()>,
         T: Trace<Key = B::Key, Val = B::Val, R = B::R, Time = <C as WithClock>::Time> + Clone,
     {
+        // ```text
```
Comment (mihaibudiu) [inline crates/dbsp/src/operator/trace.rs:238]: is this the right place for the comment, or should it be a documentation comment?

Comment (blp) [reply crates/dbsp/src/operator/trace.rs:238]: I wasn't sure. I can make it a doc comment if you think it's better.

Comment (mihaibudiu) [reply crates/dbsp/src/operator/trace.rs:238]: follow the other examples

Comment (mihaibudiu) [reply crates/dbsp/src/operator/trace.rs:238]: I don't think we have 240 diagrams. Maybe you should grep for a longer sequence of dashes.

Comment (blp) [reply crates/dbsp/src/operator/trace.rs:238]: It's not a dash, it's U+2500 BOX DRAWINGS LIGHT HORIZONTAL. We have about 239 lines with that character on comment lines.  At a glance, they are all diagrams.

Comment (mihaibudiu) [reply crates/dbsp/src/operator/trace.rs:238]: right, a diagram has many horizontal lines.
Frankly, this looks like the kind of information you would want in the documentation.

Comment (blp) [reply crates/dbsp/src/operator/trace.rs:238]: I'll move it to a doc comment.

**Pull request merged**